src/extension.ts

```diff
@@ -131,6 +132,13 @@ export async function activateInternal(ctx: vscode.ExtensionContext, perfStats:
     });
 }
 
+export async function deactivateInternal(ctx: vscode.ExtensionContext): Promise<void> {
+    await callWithTelemetryAndErrorHandling('docker.deactivate', async (activateContext: IActionContext) => {
+        activateContext.telemetry.properties.isActivationEvent = 'true';
```

Comment:
Should we consider this an activation event? (I'm not sure what we use this property for in our telemetry analysis.)

Reply:
This property is to indicate whether this telemetry is as part of extension activation and deactivation related Vs user operation.